cmd/devicesimulator/main.go

```diff
@@ -47,6 +59,8 @@ func main() {
 	initialDeviceIndex := pflag.Int("initial-device-index", 0, "starting index for device name suffix, (e.g., device-0000 for 0, device-0200 for 200))")
 	metricsAddr := pflag.String("metrics", "localhost:9093", "address for the metrics endpoint")
 	stopAfter := pflag.Duration("stop-after", 0, "stop the simulator after the specified duration")
+	versionInfo := pflag.Bool("version", false, "Print device simulator version information")
+	versionFormat := pflag.String("output", "", fmt.Sprintf("Output format. One of: (%s). Default: text format", strings.Join(outputTypes, ", ")))
```

Comment:
last nit if you use pflag.StringP you can provide short flag -o to align with cli